app/javascript/components/src/views/AddTenant/index.jsx

```diff
@@ -447,6 +447,7 @@ const AddTenant = () => {
                 showPageTitle={false}
                 handleCancel={closePropertyModal}
                 afterCreate={closeAndSetProperty}
+                showAssignPropManagers={true}
```

Comment:
Setting this to true for the modal to show the property manager selection still breaks the functionality when the list is too long. Should we just switch it to false if we don't need it here as well?

Reply:
The width of the Property Manager search field has a fixed with in `AddProperty` component. Is there a way to adjust the size of the `SearchPanel` so that it takes in the appropriate size automatically? Or we could make the width an input.

Reply:
I added a size option to the `SearchPanel` in the `AddProperty` component. So the property manager search now fits in the modal.

Reply:
The width control helps but it's the vertical size of the dropdown being off the page that's the issue:
![image](https://user-images.githubusercontent.com/8600792/163693766-2fb467ec-66aa-4742-84aa-b2d7d8e79129.png)

Reply:
I think we can limit the size across the board and make it scrollable to fix it

Reply:
I ended up using your original suggestion and removed the "Assign Property Manager". I tried setting a max height, but no matter what height I used, it always ended up pushing the "save" and "cancel" buttons off the screen.